components/fxa-client/src/lib.rs

```diff
@@ -218,6 +218,43 @@ impl FirefoxAccount {
             None => Err(ErrorKind::NoRefreshToken.into()),
         }
     }
+
+    /// Disconnect from the account and optionaly destroy our device record.
+    ///
+    /// **💾 This method alters the persisted account state.**
+    pub fn disconnect(&mut self) -> Result<()> {
+        // Start over the account state.
+        self.state.commands_data.clear();
+        self.state.device_capabilities.clear();
+        self.state.last_handled_command = None;
+        self.state.scoped_keys.clear();
```

Comment:
Nit: this feels like it could be encapsulated in a `state.clear()` helper or similar; alternately maybe we could just drop the state and re-create a fresh one?  The current setup risks forgetting to clear new state fields on disconnect.

Reply:
Thanks, I have added `start_over` functions.